tests/integration/components/conversations/conversation-part-closed-test.js

```diff
@@ -0,0 +1,46 @@
+import { moduleForComponent, test } from 'ember-qunit';
+import hbs from 'htmlbars-inline-precompile';
+import PageObject from 'ember-cli-page-object';
+import component from 'code-corps-ember/tests/pages/components/conversations/conversation-part-closed';
+import stubService from 'code-corps-ember/tests/helpers/stub-service';
+
+let page = PageObject.create(component);
+
+function renderPage() {
+  page.render(hbs`
+    {{conversations/conversation-part-closed
+      author=author
+      closedAt=closedAt
+    }}
+  `);
+
+moduleForComponent('conversations/conversation-part-closed', 'Integration | Component | conversations/conversation part closed', {
+  integration: true
+});
+
+test('if user closes message, "You closed this at" is rendered', function(assert){
+
+})
```

Comment:
These blank tests seems like they would test correct behavior, so the thinking is probably sound. Just need to implement them.